src/componets/Weather/weather.tsx

```diff
@@ -1,46 +1,44 @@
 import React from 'react';
-import GetRequestAsyncAwait from '../request/requestko'
 import Box from "./Box/Box";
 import Grid from "./Grid/Grid";
 import GridRuler from "./GridRuler/GridRuler";
+import GetWeatherData,{WeatherDataGet}from '../API/weather_query';
+import Row from '../request/test';
+
+
 
 
 function Weather() {
+  // console.log(Day())
+  let data = WeatherDataGet()
+  // let current:string = data.state
+
   return (
     <div
       className="Weather"
-      style={{ margin: "16px", position: "relative", height: "100vh" }}
+      style={{ 
+      margin: "auto", 
+      marginTop: "60px",
+      // position: "relative", 
+      height: "100vh",
+      width: "50%",
+      flex :'center',
+
+       }}
     >
-      <GridRuler spacing="sm"></GridRuler>
-
-      <Grid
-        container
-        spacing="sm"
-        alignItems="center"
-        style={{ height: "100%" }}
-      >
-        <Grid item xs={1} sm={6} md={4} lg={3}>
-          <Box>Box 1</Box>
-        </Grid>
-
-        <Grid item xs={1} sm={6} md={4} lg={3}>
-          <Box>Box 2</Box>
-        </Grid>
-
-        <Grid item xs={1} sm={6} md={4} lg={3}>
-          <Box>Box 3</Box>
-        </Grid>
-
-        <Grid item xs={1} sm={6} md={4} lg={3}>
-          <Box>Box 4</Box>
-        </Grid>
-                <Grid item xs={1} sm={6} md={4} lg={3}>
-          <Box>Box 4</Box>
-        </Grid>
-                <Grid item xs={1} sm={6} md={4} lg={3}>
-          <Box>Box 4</Box>
-        </Grid>
-      </Grid>
+
+          <Box>
+{cards?.items.map((card: ICardItem) => {
+         return <WeatherDataGet
```

Comment:
no need to use return. Return to arrow function immediately.

Reply:
nice info po salamat po